src/app/zap-templates/zcl/data-model/chip/camera-av-settings-user-level-management-cluster.xml

```diff
@@ -0,0 +1,125 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2024 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<!--
+XML generated by Alchemy; DO NOT EDIT.
+Source: src/app_clusters/CameraAVSettingsUserLevelManagement.adoc
+Parameters: in-progress
+Git: 1.4-267-gc3d95b5c9
+-->
+<configurator xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../zcl.xsd">
+  <domain name="Cameras"/>
+  <struct name="MPTZPresetStruct" apiMaturity="provisional">
+    <cluster code="0x0552"/>
+    <item fieldId="0" name="PresetID" type="int8u"/>
+    <item fieldId="1" name="Name" type="char_string" length="32"/>
+    <item fieldId="2" name="Values" type="MPTZStruct"/>
+  </struct>
+
+  <struct name="MPTZStruct" apiMaturity="provisional">
+    <cluster code="0x0552"/>
+    <item fieldId="2" name="Pan" type="int16s" optional="true" default="0" min="-180" max="180"/>
```

Comment:
This is what the spec says for field ids, but any idea why they don't just start at 0?